### DISCUSSION TITLE
containerd/cgroups package

### DISCUSSION THREAD

**Discussion by SamYuan1990** [sustainable-computing-io/kepler#558] [Q&A]
Do we consider use cgroup golang package instead of our current implementation of cgroup package?
https://github.com/containerd/cgroups

**Comment by SamYuan1990** [comment]
@rootfs , I suppose from cgroups sample as https://github.com/containerd/cgroups/blob/main/cgroup2/cpuv2_test.go for example, we may able to load cgroup data from cgroup package instead of implement by ourself?

**Comment by SamYuan1990** [comment]
https://github.com/containerd/cgroups/pull/117/files for example, for memory.current usage.

**Comment by rootfs** [comment]
@sunya-ch

**Comment by sunya-ch** [comment]
I agree that it would be great if we can use the containerd/cgroups library in our project. Just need hand to work on it :)

**Comment by marceloamaral** [comment]
+1

**Comment by SamYuan1990** [comment]
anyone wants to work on it or already working on it or we convert this into a new issue?

**Comment by marceloamaral** [comment]
@SamYuan1990, it appears that no one is currently working on this. 
If you have time, please feel free to implement this. :)

**Comment by marceloamaral** [comment]
We have addressed this in PR #609
